docs/notebook/import_data.rst

```diff
@@ -48,3 +48,19 @@ For Imviz::
     imviz = Imviz()
     imviz.app
     imviz.load_data("/path/to/data/image.fits")
+
+
+Importing Custom Line Lists
+---------------------------
+Jdaviz comes with curated line lists built from the scientific community. If you can't find the lines you need, you can add your own!::
+
+    from astropy.table import QTable
+    import astropy.units as u
+
+    lt = QTable()
```

Comment:
`lt` is too cryptic. I propose a more legible variable name.
```suggestion
    my_line_list = QTable()
```

Reply:
Good point! I did make it clearer in https://github.com/spacetelescope/aas239-jwebbinar/pull/12, so probably a good idea here too